src/app/components/view-my-rides/view-my-rides.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="container centerMe">
+    <div class="card mt-2"> 
+        <table class="table table-hover">
```

Comment:
Work on reducing code duplication by either creating a shared component for the table, or just one component for the entire view that can be reused in both places.